kapitel/kapitel1.tex

```diff
@@ -0,0 +1,54 @@
+\\\\
+
+%Angebotsspeicherung
+Beim Speichern eines Angebotes sollen alle im \ac{OTR} eingegebenen Daten, so wie, wenn anhand der Angaben möglich, berechnete Beiträge im sogennanten Maklerportal für den angemeldeten Makler gespeichert werden. Das Maklerportal ist eine Webseite an der sich Makler anmelden können wo sie Zugriff auf Tarifrechner haben, Übersichten über ihr Kundenbestand und noch vieles mehr. Zu dem können dort also auch gespeicherte Angebote dort eingesehen, bearbeitet und mit einem \ac{OTR} aufgerufen werden. Beim Speichern wird über mehrere anliegende Systeme ein Angebot in einer Datenbank gespeichert. Wichtig ist dabei die Session, welche von einem der Systeme generiert wird, realisiert als \ac{GUID}. Mithilfe dieser wird ein Angebot im XML-Format in einer Datenbank gespeichert und ist dadurch eindeutig identifizierbar.\\
+Außerdem ist es vorgesehen, dass Makler Angebote kopieren können. Dies beinhaltet die gleiche Logik wie die Angebotsspeicherung, bis auf dass eine neue \ac{GUID} erzeugt wird, wodurch dementsprechend ein neuer Eintrag in der Datenbank und im Maklerportal hinterlegt wird. Außerdem muss mit der \ac{GUID}, bzw. Session der Kopie weitergearbeitet werden.\\
+Die \ac{KM} bietet auch \ac{OTR} außerhalb des Maklerportals an. Für diese Rechner ist dann keine Angebots-, oder Kopiespeicherung vorgesehen, dementsprechend sind die Knöpfe dort nicht vorhanden. \\
+
+% Drucken
+Die Druckenfunktionalität wird durch zwei Knöpfe abgebildet, einen um ein Angebot zu drucken, der zweite für den Antrag. Ein Angebot wird z.B. dem Endkunden vom Makler unterbreitet und ein Antrag ist ein verbindliches Dokument mit welchem ein Versicherungsvertrag abgeschlossen werden kann. Somit ist diese Funktionalität sehr wichtig, als auch die Korrektheit der Daten in diesen Dokumenten. Ein Dokument soll dann gedrückt werden können, wenn Beiträge erfolgreich berechnet wurden. Dafür müssen alle berechnungsrelevanten Angaben getroffen worden sein. Die Knöpfe sind bis das geschehen ist deaktiviert und werden reaktiv aktiviert. Außerdem kann es sein, dass für einen Tarif der Angebot- oder Antragsdruck deaktiviert ist, wo dann natürlich die Knöpfe auch dementsprechend aktiv sind. Das Drucken ist dabei eigentlich das Herunterladen eines PDF-Dokumentes, damit der Makler dieses auch per E-Mail verschicken oder selber ausdrucken kann. Das Dokument wird von einem angebundenem System anhand der Daten in XML-Format generiert.
```

Comment:
- Auch das Angebot ist meines Wissens nach 4 Wochen verbindlich
- "Somit ist diese Funktionalität sehr wichtig..." ehr "Die Funktionalität wird häufig genutzt und ist damit wichtig"
- "...als auch die Korrektheit der Daten in
diesen Dokumenten" relevant für irgendetwas? Wenn nciht dann würde ich es weg nehmen
- "..., wo dann natürlich die Knöpfe auch dementsprechend aktiv sind" ergibt keinen Sinn
- "Die Knöpfe sind bis das geschehen ist deaktiviert und werden reaktiv aktiviert." würde ich in der Implementierung genauer Beschreiben, nicht im Projektziel